### ISSUE TITLE
Removal of GremlinConfiguration in PR #139 breaks the example app

### ISSUE THREAD

**Issue by fabiencoppens** [Microsoft/spring-data-gremlin#144]
The spring-data-gremlin-example app injects a `GremlinConfiguration`, a class which was removed by [PR #139 ](https://github.com/Microsoft/spring-data-gremlin/pull/139). 
How about just replacing it with a `GremlinProperties` class like it was done in `TestRepositoryConfiguration`?
I can try to push a PR tonight LA time if I have time?

**Comment by Incarnation-p-lee** [comment]
Yes, we refine this as there may be cycle dependency in previous. And the sample code may not update respectively. Thank you for preparing the PR for example.
